Fedya/FANUC Client/RobotComunicationDifinition.inl

```diff
@@ -133,22 +109,30 @@ void RobotConnect<T>::sendParalel(std::mutex* mt, bool* f, RobotConnect* ins)
 }
 
 template <typename T>
-int RobotConnect<T>::startWorking()
+int RobotConnect<T>::beginConnection()
 {
-	const int iResult = conRobot();
-	if (iResult)
+	//�������� ������ ��������
+	_sockSend = SocketWorking::getInstance().getConnectedSocket(_serverAddrString.c_str(), _sendPort, _timeBetweenReconnection);
+	if (_sockSend == INVALID_SOCKET)
 	{
-		//std::cout << "Conection error " << iResult << " " << WSAGetLastError() << std::endl;
-		myInterface::MyShower::getInstance().addLog("Conection error ", WSAGetLastError());
+		myInterface::MyShower::getInstance().addLog("send socket error ", WSAGetLastError());
+		return 1;
+	}
+
+	//������� ������ ������
+	_sockRecv = SocketWorking::getInstance().getConnectedSocket(_serverAddrString.c_str(), _recvPort, _timeBetweenReconnection);
+	if (_sockRecv == INVALID_SOCKET)
+	{
+		myInterface::MyShower::getInstance().addLog("recive socket error ", WSAGetLastError());
 		closesocket(_sockSend);
-		closesocket(_sockRecv);
-		return -4;
+		return 2;
 	}
+
 	/// ���������� � ������
 
 	//type of cordiates:0 = JOINT | 1 = JOGFRAME | 2 = WORLDFRAME | 3 = TOOLFRAME | 4 = USER FRAME
 	char coord[3] = { 0,0,0 };
-	sprintf_s(coord, "%d ", _syscoord);
+	sprintf_s(coord, "%d ", _sysCoord);
```

Comment:
**(must be change)**: change this code to `std::to_string()` and `std::string::c_str()`.